### DISCUSSION TITLE
Stanford CS148 - CS自学指南

### DISCUSSION THREAD

**Discussion by giscus** [PKUFlyingPig/cs-self-learning#198] [Announcements]
# Stanford CS148 - CS自学指南

CS自学指南

https://csdiy.wiki/%E8%AE%A1%E7%AE%97%E6%9C%BA%E5%9B%BE%E5%BD%A2%E5%AD%A6/CS148/

**Comment by Chenmingwang** [comment]
###Great!

**Reply by kayllcode** [reply]
非常感谢！

**Comment by Reno4205** [comment]
刚从推特发现了这个宝藏~目前在学python for everybody 自学没有deadline的确对自律要求很高！共勉

**Reply by BlueCompactDwarf** [reply]
我也在看py4e，请问是不是看完了py4e再来学CS61A会比较好呢，本人没有任何基础

**Reply by rainy-xie** [reply]
不会，因为CS61A里本身就带简明的pyhton教程

**Comment by PLANETDUST** [comment]
本人一个差不多算计算机新手，不聪明的一个普通人今天开始认真自学，不知道能不能学懂学会，能不能一直坚持下去。希望我可以。

**Reply by electrojay27** [reply]
一起一起

**Comment by Egg12138** [comment]
本科居然能扎实刷完这么多课程，作者太NB了

**Comment by SoulXXXX** [comment]
作者是在本科阶段就学完这些东西了吗，太强了吧

**Comment by barbarianxs** [comment]
跨考过来的打算学一下cs61b和cs186，关于数据结构和数据库的

**Reply by wangcham** [reply]
有兴趣一块学啊🌝

**Comment by Summery36** [comment]
准大一来预习啦

**Reply by Summery36** [reply]
有没有一起学的，我们加个微信建群互相监督，一起学！我的微信号是：zmf1127232045

**Comment by ogoattt** [comment]
Fatastic!

**Comment by dddjjjaaa** [comment]
按流程，先膜拜，再阅读

**Comment by Chunjie-DAI** [comment]
真是太棒了！

**Comment by Summery36** [comment]
英语教学会不会影响学习效率啊？

**Reply by violet1123** [reply]
越早适应英语对自己益处越大，英文的优质资源太多太多

**Comment by yinyihong** [comment]
留下我的爪，立志成为一个数据科学家。

**Comment by Zacharia2** [comment]
真的是，太太太厉害了，留下我的爪印
。

**Comment by hyu17** [comment]
现在开始，坚持学习!

**Comment by wangcham** [comment]
大二开学知道这个网站，作者大大厉害啊

**Reply by OscarLight77** [reply]
俺也一样，大二开始搞

**Comment by electrojay27** [comment]
我才大一，对这些还不了解，只能慢慢来了



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2022年9月5日(周一) 下午4:52
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [PKUFlyingPig/cs-self-learning] Stanford CS148 - CS自学指南 (Discussion #198)




 
俺也一样，大二开始搞

**Comment by electrojay27** [comment]
主要学习英语我没有很有效的方法



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2022年9月22日(周四) 下午4:07
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [PKUFlyingPig/cs-self-learning] Stanford CS148 - CS自学指南 (Discussion #198)

**Comment by electrojay27** [comment]
可以啊



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2022年10月11日(周二) 下午3:58
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [PKUFlyingPig/cs-self-learning] Stanford CS148 - CS自学指南 (Discussion #198)




 
有兴趣一块学啊🌝
